### DISCUSSION TITLE
使用-6参数ipv6无法来连接

### DISCUSSION THREAD

**Discussion by soulherats** [opengnb/opengnb#42] [Q&A] [answered]
两边分别执行 gnb  -n 1001 -I "101.32.178.3|9001" -p xx -e "--wan-address6-file=/tmp/wan6_addr.dump" -6 -V与gnb  -n 1001 -I "101.32.178.3|9001" -p xx -e "--wan-address6-file=/tmp/wan6_addr.dump" -6 -V 
无法ping通，有没有大神提供个支持ipv6的node
24-09-11 20:54:03 CORE GNB version Dev 1.5.0.c  protocol version 1.5.0
24-09-11 20:54:03 CORE Copyright (C) 2019 gnbdev<gnbdev@qq.com>
24-09-11 20:54:03 CORE Site: https://github.com/gnbdev/opengnb
24-09-11 20:54:03 CORE Build Time [Sep 11 2024,20:27:38]
24-09-11 20:54:03 NODE gnb_node_worker init finish
24-09-11 20:54:03 INDEX gnb_index_worker init finish
24-09-11 20:54:03 FULL_DETECT gnb_detect_worker init finish
24-09-11 20:54:03 INDEX_SERVICE gnb_index_service_worker init finish
24-09-11 20:54:03 PF gnb_pf_route init
24-09-11 20:54:03 PF gnb_pf_crypto_xor init
24-09-11 20:54:03 MAIN gnb_primary_worker init finish
24-09-11 20:54:03 CORE gnb core created!
24-09-11 20:54:03 CORE Start.....
24-09-11 20:54:03 CORE if[gnb_tun] opened
24-09-11 20:54:03 CORE node[1001] ipv4[10.1.0.1]
24-09-11 20:54:03 CORE gnb_index_worker start
24-09-11 20:54:03 CORE gnb_detect_worker start
24-09-11 20:54:03 CORE gnb_index_service_worker start
24-09-11 20:54:03 CORE gnb_node_worker start
24-09-11 20:54:03 MAIN bind addr6 [:::9001]
24-09-11 20:54:03 CORE gnb_primary_worker start
24-09-11 20:54:03 NODE start gnb_node_worker success!
24-09-11 20:54:03 MAIN start gnb_primary_worker success!
24-09-11 20:54:03 INDEX_SERVICE start gnb_index_service_worker success!
24-09-11 20:54:03 INDEX start gnb_index_worker success!
24-09-11 20:54:03 FULL_DETECT start gnb_detect_worker success!
24-09-11 20:54:03 INDEX post local address to index ***.32.178.3:9001
24-09-11 20:54:04 CORE exec gnb_es argv '/usr/bin/gnb_es --wan-address6-file=/tmp/wan6_addr.dump -b /tmp/gnb.9001.map --if-loop --upnp'
24-09-11 20:55:04 INDEX post local address to index ***.32.178.3:9001
24-09-11 20:56:05 INDEX post local address to index ***.32.178.3:9001
24-09-11 20:57:06 INDEX post local address to index ***.32.178.3:9001
24-09-11 20:58:07 INDEX post local address to index ***.32.178.3:9001
24-09-11 20:59:05 CORE exec gnb_es argv '/usr/bin/gnb_es --wan-address6-file=/tmp/wan6_addr.dump -b /tmp/gnb.9001.map --if-loop --upnp'
24-09-11 20:59:08 INDEX post local address to index ***.32.178.3:9001
24-09-11 21:00:09 INDEX post local address to index ***.32.178.3:9001
24-09-11 21:01:10 INDEX post local address to index ***.32.178.3:9001

**Comment by gnbdev** [comment]
既然选用了 -6

**Reply by soulherats** [reply, marked as answer]
不是说支持ipv上报么

**Reply by gnbdev** [reply]
> 不是说支持ipv上报么
是支持，-6 禁用了ipv4，你用的是公共节点是ipv4。

**Reply by soulherats** [reply]
那就是说我这里要填公共节点是ipv6地址了；另外不指定-6参数如果公共节点没有ipv6地址也是没办法上报ipv6地址的吗
